cmd/sandbox-ce/compose/compose.go

```diff
@@ -2,16 +2,117 @@ package compose
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
 )
 
-func Run(ctx context.Context, arg ...string) error {
-	cmd := exec.CommandContext(ctx, "docker-compose", arg...)
+const composeContainerPath = "https://github.com/docker/compose/releases/download/1.24.0/run.sh"
+
+var envKeys = map[string]bool{
+	"GITBASE_REPOS_DIR": true,
+}
+
+type Compose struct {
+	bin string
+}
+
+func (c *Compose) Run(ctx context.Context, arg ...string) error {
+	cmd := exec.CommandContext(ctx, c.bin, arg...)
+
+	var compOpts []string
+	for _, keyValue := range os.Environ() {
```

Comment:
why no [LookupEnv](https://golang.org/pkg/os/#LookupEnv)?

Reply:
Simply because I didn't know it 😂
Thanks for the tip!

Reply:
Addressed [here](https://github.com/smacker/superset-compose/pull/4/commits/5646ab04f2b558d5bb64d139716bb39e95476f70).